megamek/src/megamek/server/Server.java

```diff
@@ -27786,49 +26771,46 @@ private void applyCargoCritical(Aero aero, int damageCaused, Vector<Report> repo
      * Rolls and resolves critical hits with a die roll modifier.
      */
 
-    public Vector<Report> criticalEntity(Entity en, int loc, boolean isRear,
-                                         int critMod, int damage) {
+    public Vector<Report> criticalEntity(Entity en, int loc, boolean isRear, int critMod, int damage) {
         return criticalEntity(en, loc, isRear, critMod, true, false, damage);
     }
 
     /**
      * Rolls and resolves critical hits with a die roll modifier.
      */
 
-    public Vector<Report> criticalEntity(Entity en, int loc, boolean isRear,
-                                         int critMod, int damage, boolean damagedByFire) {
+    public Vector<Report> criticalEntity(Entity en, int loc, boolean isRear, int critMod, int damage,
+                                         boolean damagedByFire) {
         return criticalEntity(en, loc, isRear, critMod, true, false, damage, damagedByFire);
     }
 
     /**
      * Rolls one critical hit
      */
-    public Vector<Report> oneCriticalEntity(Entity en, int loc,
-                                             boolean isRear, int damage) {
+    public Vector<Report> oneCriticalEntity(Entity en, int loc, boolean isRear, int damage) {
         return criticalEntity(en, loc, isRear, 0, false, false, damage);
     }
 
     /**
      * Rolls one critical hit
      */
-    public Vector<Report> oneCriticalEntity(Entity en, int loc,
-                                             boolean isRear, int damage, boolean damagedByFire) {
+    // TODO : Windchild can this be removed? - never called
+    public Vector<Report> oneCriticalEntity(Entity en, int loc, boolean isRear, int damage,
```

Comment:
Can this be removed? It is never called

Reply:
I don't see any need to maintain unused versions of overloaded methods.